deploy/config/dashboard.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# dashboard server side global configuration
+
+http:
+  hostname: localhost
+  port: 8000
+  favicon: src/assets/favicon.ico
+  static:
+    /assets: src/assets
+    development:
+      /build: build
+    production:
+      /dist: dist
+
+serverUrl: http://openpitrix-api-gateway:9100
```

Comment:
i think you only need `serverUrl` and `apiVersion`, then web app read it from env